pytorch_blade/pytorch_blade/compiler/mlir/converters/torch_mlir_op_filter.cpp

```diff
@@ -38,6 +38,9 @@ bool IsTorchMlirSupported(const torch::jit::Node& node) {
 // clang-format off
 const std::unordered_set<std::string> &GetTorchMlirWhiteList() {
   static std::unordered_set<std::string> white_list{
+#if defined(PLATFORM_ALIBABA) and defined(ENABLE_BLADE_GEMM)
+      "aten::upsample_nearest2d",
```

Comment:
Why not add aten::upsample_nearest2d.vec?

Reply:
OK.